### PR TITLE
Updated documentation of pipelines around approvals

### PR DESCRIPTION
**Pull request by mswiderski**
# Changes

Updated pipelines documentation, known custom tasks to include approvals that have support for both `v1beta1.CustomRun` and `v1alpha1.Run`

/kind documentation

# Submitter Checklist

As the author of this PR, please check off the items in this checklist:

- [ ] Has [Docs](https://github.com/tektoncd/community/blob/main/standards.md#docs) included if any changes are user facing
- [ ] Has [Tests](https://github.com/tektoncd/community/blob/main/standards.md#tests) included if any functionality added or changed
- [ ] Follows the [commit message standard](https://github.com/tektoncd/community/blob/main/standards.md#commits)
- [ ] Meets the [Tekton contributor standards](https://github.com/tektoncd/community/blob/main/standards.md) (including
  functionality, content, code)
- [ ] Has a kind label. You can add one by adding a comment on this PR that contains `/kind <type>`. Valid types are bug, cleanup, design, documentation, feature, flake, misc, question, tep
- [ ] Release notes block below has been updated with any user facing changes (API changes, bug fixes, changes requiring upgrade notices or deprecation warnings)
- [ ] Release notes contains the string "action required" if the change requires additional action from users switching to the new release

# Release Notes

```release-note
NONE
```

### CODE REVIEW

**Comment by jerop** [comment]
@mswiderski thanks for sharing the approvals custom task beta, and happy to hear you've migrated! is there any feedback you can share? it'd be helpful as we work towards a v1 release of custom tasks 

/approve 
/ok-to-test

File: docs/pipelines.md [hunk 1]
```diff
@@ -1819,6 +1819,7 @@ We try to list as many known Custom Tasks as possible here so that users can eas
 | Custom Task                      | Description                                                                                                                      |
 |:---------------------------------|:---------------------------------------------------------------------------------------------------------------------------------|
 | [Wait Task Beta][wait-task-beta] | Waits a given amount of time before succeeding, specified by an input parameter named duration. Support `timeout` and `retries`. |
+| [Approvals][approvals-beta]| Pauses the execution of `PipelineRuns` and waits for manual approvals. Bersion 0.6.0 and up. |
```
Comment (jerop) [inline docs/pipelines.md:1822]:
```suggestion
| [Approvals][approvals-beta]| Pauses the execution of `PipelineRuns` and waits for manual approvals. Version 0.6.0 and up. |
```

**Comment by jerop** [comment]
please squash the commits - we prefer one commit per pull request 

https://github.com/tektoncd/community/blob/main/standards.md#small-pull-requests 
https://github.com/tektoncd/community/blob/main/standards.md#commit-messages

**Comment by mswiderski** [comment]
@jerop done, thanks for quick feedback

**Comment by XinruZhang** [comment]
/lgtm

Thank you for the update @mswiderski ❤️

**Comment by mswiderski** [comment]
/retest

not sure these test failures are related so could use some help in assessing it

**Pull request merged**